### PR TITLE
added my name

### PR DESCRIPTION
**Pull request by cheeseburgur101**
:D

### CODE REVIEW

**Comment by georgetzavelas** [comment]
Cool!

**Comment by cheeseburgur101** [comment]
thanks!

**Comment by cheeseburgur101** [comment]
ok!

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# test
\ No newline at end of file
+# test
+/*Hello World from Nathan G*/
```
Comment (georgetzavelas) [inline README.md]: Hello FRC instead of Hello World

**Pull request merged**